### DISCUSSION TITLE
Any way to prevent Screenshots in v-img?

### DISCUSSION THREAD

**Discussion by arauko** [vuetifyjs/vuetify#14220] [Q&A] [answered]
Hello,
My client asks me if there would be any way to prevent users to take a screenshot of the web or recording, since the content uploaded to that website is owned by each user, and we have a subscription system to access that content, we're not interested that another user can "steal" that content and publish it on other websites. 

We are interested in Avoid the screen capture or Show black screen on the v-img component

Since we are using vuetify, I was wondering if there is something similar with v-img.

thanks and best regards

**Comment by jacekkarczmarczyk** [comment, marked as answer]
There is no way to do this

**Reply by KaelWD** [reply]
Even netflix and amazon can't prevent their videos from being copied.

**Reply by arauko** [reply]
yeah, I thought so, I just needed to make sure

thank you very much.
